tests/common/dualtor/control_plane_utils.py

```diff
@@ -85,5 +96,27 @@ def expect_db_values(duthost, db, state, health=None, intf_names='all'):
             if db_dump[table_key]['value'][field] != target_value:
                 mismatch_ports[table_key] = db_dump[table_key]['value']
 
-    pytest_assert(not bool(mismatch_ports), "Database states don't match expected state {state}, incorrect {db_name} values {db_states}"
-                                                    .format(state=state, db_name=DB_NAME_MAP[db], db_states=json.dumps(mismatch_ports, indent=4, sort_keys=True)))
\ No newline at end of file
+    pytest_assert(not bool(mismatch_ports),
+                  "Database states don't match expected state {state},"
+                  "incorrect {db_name} values {db_states}"
+                  .format(state=state, db_name=DB_NAME_MAP[db],
+                          db_states=json.dumps(mismatch_ports,
+                                               indent=4,
+                                               sort_keys=True)))
+
+
+def verify_tor_states(expected_active_host, expected_standby_host,
+                      expected_standby_health='healthy', intf_names='all'):
+    """
+    Verifies that the expected states for active and standby ToRs are
+    reflected in APP_DB and STATE_DB on each device
+    """
+
+    expect_db_values(expected_active_host, APP_DB,
+                     'active', intf_names=intf_names)
+    expect_db_values(expected_active_host, STATE_DB, 'active',
+                     'healthy', intf_names=intf_names)
+    expect_db_values(expected_standby_host, APP_DB,
+                     'standby', intf_names=intf_names)
+    expect_db_values(expected_standby_host, STATE_DB, 'standby',
+                     expected_standby_health, intf_names=intf_names)
```

Comment:
Was the line at EOF removed?

Reply:
There is a blank line, but Github doesn't display it. It displays the ⛔ symbol if it's not present I think